test/attr.test.js

```diff
@@ -55,11 +55,11 @@ describe('#attr', () => {
     it('fails when the actual does not match the expected', (wrapper) => {
       expect(() => {
         expect(wrapper).to.have.attr('id', 'invalid')
-      }).to.throw(`to have a 'id' attr`)
+      }).to.throw("to have a 'id' attr")
```

Comment:
Standard uses eslint in the background, reason why we decided to go with it is that we don't want to lose valuable time discussing code style. Mission failed 😄 While I (& others) don't agree with everything, let's just accept the rules as is and move on. If the rule set bothers us a lot in the future we can re-assess